package.json

```diff
@@ -12,11 +12,18 @@
   "author": "ljahn, slazar, mleitgeb",
   "license": "GPL-3.0-or-later",
   "dependencies": {
+    "axios": "^1.6.4",
+    "bcrypt": "^5.1.1",
+    "body-parser": "^1.20.2",
+    "cors": "^2.8.5",
```

Comment:
Did we need that for anything, or did you just want to increase security?

Reply:
it's not needed in this feature, but it's needed for when you send requests from the frontend server. I was testing some things and forgot to remove it when I finished